go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/multiversx/twofactor
+
+go 1.17
```

Comment:
in multi-factor-auth repo, for example, we changed to 1.18, maybe change here also

Reply:
updated